packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -75,9 +80,36 @@ const StyledNativeSelect = styled.select<{
   }
 `;
 
+const StyledFormFieldContainer = styled(FormFieldContainer)<{
+  additionalContent?: React.ReactNode;
+  hasLabel?: boolean;
+  labelPosition?: LabelPosition;
+}>`
+  display: ${props => (props.labelPosition === 'left' ? 'flex' : '')};
+  flex: ${props => (props.additionalContent ? '1' : '')};
+  label {
+    display: ${props => (props.additionalContent ? 'flex' : '')};
+    justify-content: ${props =>
+      props.additionalContent ? 'space-between' : ''};
+    align-items: ${props => (props.additionalContent ? 'center' : '')};
+  }
```

Comment:
I wonder if this would be cleaner if we checked for `props.additionalContent` at the top level, then you can just add the styles without all the ternaries.